src/main/java/uk/gov/hmcts/reform/jobscheduler/jobs/HttpCallJob.java

```diff
@@ -4,30 +4,49 @@
 import org.quartz.JobExecutionContext;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
 import uk.gov.hmcts.reform.jobscheduler.model.HttpAction;
-import uk.gov.hmcts.reform.jobscheduler.services.jobs.ActionSerializer;
 
 @Component
 public class HttpCallJob implements Job {
 
     public static final String PARAMS_KEY = "params";
     private static final Logger logger = LoggerFactory.getLogger(HttpCallJob.class);
 
-    private final ActionSerializer serializer;
+    private final RestTemplate restTemplate;
```

Comment:
Not sure if important, but `RestTemplate` is an implementation of `RestOperations`, which is actually used. Would it be worthy replacing? I'll leave it to your judgement.